lib/BackgroundJob/Tasks/AddMissingImagesTask.php

```diff
@@ -161,8 +162,7 @@ private function getPicturesFromFolder(Folder $folder, $results = array()) {
 			if ($node instanceof Folder and !$node->nodeExists('.nomedia')) {
 				$results = $this->getPicturesFromFolder($node, $results);
 			} else if ($node instanceof File) {
-				// todo: which are filetypes we can work with
-				if ($node->getMimeType() === 'image/jpeg') {
+				if (Requirements::isImageTypeSupported($node->getMimeType())) {
```

Comment:
Do you know how to use dependency injection in these cases?.
Honestly did not try ..

Reply:
Dependency injection when using static methods? Not possible:D (static methods are antipattern for DI).
If we ever plan to mock "isImageTypeSupported", it will be impossible. However, I don't see us using different implementation of this method. If we ever do, we should just make it non-static, I guess